pkg/cache/cache.go

```diff
@@ -724,20 +724,20 @@ func (c *Cache) ClusterQueueAncestors(cqObj *kueue.ClusterQueue) ([]kueue.Cohort
 	}
 
 	cohort := c.hm.Cohort(cqObj.Spec.Cohort)
-	if cohort == nil {
+	if cohort == nil || !cohort.HasParent() {
```

Comment:
nit - code below hadles this case well, by returning empty list when singleton Cohort

```suggestion
	if cohort == nil {
```

wil just have to update unit test to use cmp opt, EquateEmpty

Reply:
Ack, I wasn't sure if empty list would be acceptable